File: JavaScript/1672. Richest Customer Wealth.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(accounts) {
    var a = 0;
    accounts.forEach(element => {
        var b = 0;
        element.forEach(val => {
            b = b + val;
        });
        if(b > a) a = b;
    });
    return a;
};

File: Java/1672. Richest Customer Wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int ans = 0, temp = 0;

        for(int[] i: accounts){
            temp = 0;
            for(int j: i) temp += j;
            ans = Math.max(ans, temp);
        }

        return ans;
    }
}
